documentation/inspecting_plans.md

```diff
@@ -22,15 +30,31 @@ plans](./bolt_running_plans.md#how-bolt-locates-plans)
 ## Show documentation for a plan
 
 Use the following command to view parameters and other details for a plan,
-including whether a plan supports `--noop`:
+including whether a plan supports `noop`:
+
+- _\*nix shell command_
+
+  ```shell
+  bolt plan show <PLAN NAME>
+  ```
 
-```
-bolt plan show <PLAN_NAME>
-```
+- _PowerShell cmdlet_
+
+  ```powershell
+  Get-BoltPlan -Name <PLAN NAME>
+  ```
 
 For example, to see the parameters and documentation for the `facts::info` plan,
 run:
 
-```
-bolt plan show facts::info
-```
\ No newline at end of file
+- _\*nix shell command_
+
+  ```shell
+  bolt plan show facts::info
```

Comment:
It seems a bit overkill to show both `bolt plan show <PLAN NAME>` and `bolt plan show facts::info`. We don't seem to have the latter kind of example for tasks (only `bolt task show <PLAN NAME>`)